### PR TITLE
Ansible: SSH into shell servers as root

### PR DESCRIPTION
**Pull request by KellerFuchs**
This is a workaround for hashbang/shell-etc#74

### CODE REVIEW

**Comment by KellerFuchs** [comment]
@hashbang/administrators Review needed.

**Review by drGrove — APPROVED** [review]

File: hosts [hunk 1]
```diff
@@ -4,6 +4,10 @@ ny1.hashbang.sh
 sf1.hashbang.sh
 to1.hashbang.sh
 
+[shell_servers]
+ansible_user=root
```
Comment (daurnimator) [inline hosts:8]: Can you add a link to issue 74 in a comment here?

Comment (KellerFuchs) [reply hosts:8]: Arg.  Will do... in a separate PR?  :V

In general, though, there is no benefit not to do this: direct root isn't any more privileged than admin+sudo, and this avoids the dependency on LDAP/userdb.

Comment (daurnimator) [reply hosts:8]: In general I like to disable root login via sshd. I'd like to do that for hashbang too but... this. :)

Comment (KellerFuchs) [reply hosts:8]: Eeeh, experience (#! and otherwise) says that having a method of getting root without relying on LDAP is good  :3

**Pull request merged**